### DISCUSSION TITLE
[BUG] BCC mode no include function and exclude doesn't work

### DISCUSSION THREAD

**Discussion by MickCodes** [dashingsoft/pyarmor#1433] [General]
I have a larger file, with over 20,000 lines of code, and just recently purchased Pyarmor Pro but am struggling to get any of the pyarmor pro features to work.

RFT Enabled, pyarmor is stuck at parsing the script for a very long time and not working

BCC, Excluding/Including functions isnt working. BCC Mode also has issues with any async functions, which im trying to exclude but cant and when i want to include specific functions that also doesnt work, pyarmor simply ignores any inclusions/exclusions.

I am also trying to do this via the pyarmor webui, does adding these exclusions/inclusions work when using the webui? if not, could you support this please.

*suggestion* make an include only for rft/bcc, to only include certain names without touching any of the other code.

**Comment by MickCodes** [comment]
I would also like to add, I believe pyarmor is also ignoring any inclusion/exclusions for RFT mode

**Comment by jondy** [comment]
duplicated #1428

It will be fixed in next release.

**Comment by jondy** [comment]
Refer to
https://pyarmor.readthedocs.io/en/latest/topic/bccmode.html#ignore-module-or-function
